packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -30,8 +30,8 @@ export interface MemberVideoPane {
 }
 
 export interface VideoLayout {
-  screenShareVideo: {
-    size: PaneSize | null; // null if you don't want to receive any screen share video
+  screenShareVideo?: {
```

Comment:
I've decided to make this optional so that people don't have to mention screenShareVideo at all in layouts that don't need it